azure-pipelines/azure-pipelines.yml

```diff
@@ -443,7 +443,7 @@ stages:
         displayName: build edgehub/deviceshifu-http-plc4x
 
 - stage: docker_build_muiltiarch_and_push
-  condition: and(succeeded(), startsWith(variables['Build.SourceBranch'], 'refs/tags/'))
+  condition: and(succeeded(), ne(variables['Build.Reason'], 'PullRequest'), not(contains(variables['Build.SourceBranchName'],"rc")))
```

Comment:
need to revert changes for docker push, we only need this condition for demo tar releases